client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -65,7 +65,11 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 	const siteTitle = site?.name;
 	const isReskinned = true;
 	const isAtomic = useSelect( ( select ) => site && select( SITE_STORE ).isSiteAtomic( site.ID ) );
-	const isPrivateAtomic = Boolean( site?.launch_status === 'unlaunched' && isAtomic );
+	useEffect( () => {
```

Comment:
Will it be better if we just remove this step for the atomic site in https://github.com/Automattic/wp-calypso/blob/af74157985fb4eda9f1ddfd310285a7427f3837a/client/landing/stepper/declarative-flow/site-setup-flow.ts#L35?

Reply:
ahh good idea!

Reply:
Actually, this didn't work as I'd hoped.

simply removing 'designSetup' from the list for atomic sites caused the user to be redirected back to the 'verticals' step in a loop.

I also investigated moving the check for atomic sites into the `useAssertConditions` hook, but I couldn't use the 'exitFlow' function which would allow me to keep the `write` flow working on atomic sites but exit the flow if they attempt to use the design picker.

Reply:
> simply removing 'designSetup' from the list for atomic sites caused the user to be redirected back to the 'verticals' step in a loop.

That sounds bad 😞 And we have to handle the navigation in this way which is not what I expected. Let's go with current solution 👍